contracts/test/_fixture-sonic.js

```diff
@@ -109,6 +109,8 @@ const defaultSonicFixture = deployments.createFixture(async () => {
   const strategist = await impersonateAndFund(strategistAddr);
   strategist.address = strategistAddr;
 
+  await sonicStakingStrategy.connect(strategist).setDefaultValidatorId(18);
```

Comment:
I think this should be in the deploy script.

Reply:
I see why its in the fixture. The governor can not set the default

Reply:
Yes we need to do it manually with the 2/8 multisig